palm/cli.py

```diff
@@ -90,18 +90,12 @@ def get_version():
 def required_dependencies_ready():
 
     docker_checks = (
-        (
-            "docker --version",
-            "Docker is not installed, please install it first",
-        ),
+        ("docker --version", "Docker is not installed, please install it first",),
         (
             "docker-compose --version",
             "Docker Compose is not installed, please install it first",
         ),
-        (
-            "docker ps",
-            "Docker is not running, please start it first",
-        ),
+        ("docker ps", "Docker is not running, please start it first",),
```

Comment:
I agree this looks better... but `black` does not. Linting ablaze with this change